packages/html2react/src/libraries/component.tsx

```diff
@@ -1,20 +1,23 @@
 /* eslint-disable @typescript-eslint/no-use-before-define */
 import React from "react";
-import { connect, error } from "frontity";
+import { connect, error, warn } from "frontity";
 import { Connect } from "frontity/types";
 import Html2ReactType, {
   Component,
   HandleNodes,
   HandleNode,
-  ApplyProcessors,
-  OldProcessor,
-  NewProcessor
+  ApplyProcessors
 } from "../../types";
 
 const applyProcessors: ApplyProcessors = ({ node, processors, ...payload }) => {
   for (const proc of processors) {
-    const processor =
-      (proc as NewProcessor).processor || (proc as OldProcessor).process;
+    // Add deprecation warning for process.
+    if ((proc as any).process)
+      warn(
+        "The property 'process' has been deprecated. Please rename it to 'processor' instead."
```

Comment:
Just a small nitpick: I think I would word it a bit differently. Can we add a link to the docs as well? Something like:

`The property 'process' has been deprecated. Please use the new 'processor' property instead. See the documentation https://docs.frontity.org/api-reference-1/frontity-html2react#create-your-own-processors for details about the new API`.

Just to make it clear that the API has changed and not ONLY the name.

Reply:
I was waiting until we have a migration guide for this but I agree that linking to the new documentation is good enough. I'll do that, thanks @michalczaplinski.